sdcm/wait.py

```diff
@@ -30,14 +65,13 @@ def wait_for(func, step=1, text=None, *args, **kwargs):
     :param func: Function to evaluate.
     :param step: Amount of time to sleep before another try.
     :param text: Text to log, for debugging purposes.
-    :param args: Positional arguments to func
     :param kwargs: Keyword arguments to func
```

Comment:
:param kwargs: Keyword arguments to func, contains both converted non-keyword arguments and original keyword arguments. Example: foo(arg1, arg2=None) -> forever_wait_for(foo, arg1='v1', arg2='v2')

Reply:
This code will work

Reply:
Current code works. If people who uses wait_for() in future has concern about passing non-keyword arguments of func, they can find helper by above description.